src/custom/pages/Faq/index.tsx

```diff
@@ -326,20 +349,116 @@ export default function Faq() {
-              Ethereum address that will identify the solver.
+              Once the GNO tokens have been staked (locked up), GnosisDAO has to vote to approve or reject the Ethereum
+              address that will identify the solver.
             </li>
             <li>
               Additionally, a solver must have the technical knowledge to create the appropriate batch settlement
               solutions, or take the risk of being slashed by the GnosisDAO for wrongdoing.
             </li>
           </ol>
+
+          <h3 id="what-interactions-can-i-encounter-when-using-Cowswap">
+            What interactions can I encounter when using CowSwap?
+          </h3>
+
+          <p>
+            <strong>Internal CowSwap Operations</strong>
+          </p>
+          <div id="table-container">
+            <table>
+              <thead>
+                <tr>
+                  <th>Action</th>
+                  <th>Description</th>
+                  <th>Costs</th>
+                  <th>Action Performed</th>
+                  <th>Pay for gas?</th>
+                  <th>What do I need to pay the gas costs with?</th>
+                </tr>
+              </thead>
+              <tbody>
+                <tr>
+                  <td>Approve token for trading</td>
+                  <td>
+                    Required step for being able to sell a token.Only needs to be done once. Afterward, you will be able
+                    to trade the token using gasless transactions!
+                  </td>
+                  <td>Regular Ethereum Tx.</td>
+                  <td>Set the token allowance for the Allowance Manager</td>
+                  <td>Yes</td>
+                  <td>ETH</td>
+                </tr>
+                <tr>
+                  <td>Signing an Order</td>
+                  <td>
+                    Signature of a gasless off-chain order. You define your limit price and expiration date. The order
+                    will try to be executed using MEV protection against different on-chain liquidity sources or other
+                    CowSwap users trading in the same block.
+                  </td>
+                  <td>Free</td>
+                  <td>None</td>
+                  <td>No</td>
+                  <td>Free</td>
+                </tr>
+                <tr>
+                  <td>Trade</td>
+                  <td>
+                    The trade will happen automatically if a CoW (Coincidence of Wants) is found, or if the order can be
+                    executed at the specified price in any on-chain liquidity source. You only pay fees if the trade is
+                    successful!
+                  </td>
+                  <td>Gnosis Protocol Fee which is paid either in your sell or buy token.</td>
+                  <td>Accept the trade details by executing your limit order</td>
+                  <td>No</td>
+                  <td>Paid in either Sell or Buy token</td>
```

Comment:
I might be mistaken but I think I already pointing this out somewhere else.
We only charge in the sell token, even though it might look like we don't.
Shouldn't we say so here?

Reply:
Ah, I was not aware of this @alfetopito . @anxolin made the table, so would be great to get his opinion